packages/renderer/src/lib/kube/KubernetesDashboard.svelte

```diff
@@ -0,0 +1,143 @@
+<script lang="ts">
+import { Link, NavPage } from '@podman-desktop/ui-svelte';
+import { onDestroy, onMount } from 'svelte';
+import type { Unsubscriber } from 'svelte/store';
+
+import KubernetesCurrentContextConnectionBadge from '/@/lib/ui/KubernetesCurrentContextConnectionBadge.svelte';
+import {
+  kubernetesCurrentContextConfigMaps,
+  kubernetesCurrentContextDeployments,
+  kubernetesCurrentContextIngresses,
+  kubernetesCurrentContextNodes,
+  kubernetesCurrentContextPersistentVolumeClaims,
+  kubernetesCurrentContextRoutes,
+  kubernetesCurrentContextSecrets,
+  kubernetesCurrentContextServices,
+  kubernetesCurrentContextState,
+} from '/@/stores/kubernetes-contexts-state';
+import { NO_CURRENT_CONTEXT_ERROR } from '/@api/kubernetes-contexts-states';
+
+import { kubernetesContexts } from '../../stores/kubernetes-contexts';
+import ConfigMapSecretIcon from '../images/ConfigMapSecretIcon.svelte';
+import DeploymentIcon from '../images/DeploymentIcon.svelte';
+import IngressRouteIcon from '../images/IngressRouteIcon.svelte';
+import NodeIcon from '../images/NodeIcon.svelte';
+import PvcIcon from '../images/PVCIcon.svelte';
+import ServiceIcon from '../images/ServiceIcon.svelte';
+import KubeApplyYamlButton from './KubeApplyYAMLButton.svelte';
+import KubernetesDashboardGuideCard from './KubernetesDashboardGuideCard.svelte';
+import KubernetesDashboardResourceCard from './KubernetesDashboardResourceCard.svelte';
+import KubernetesEmptyPage from './KubernetesEmptyPage.svelte';
+
+let welcome = $derived($kubernetesCurrentContextState.error === NO_CURRENT_CONTEXT_ERROR);
+let currentContextName = $derived($kubernetesContexts.find(context => context.currentContext)?.name);
+
+let nodeCount = $state<number>(0);
+let deploymentCount = $state<number>(0);
+let serviceCount = $state<number>(0);
+let ingressCount = $state<number>(0);
+let routeCount = $state<number>(0);
+let pvcCount = $state<number>(0);
+let configMapCount = $state<number>(0);
+let secretCount = $state<number>(0);
```

Comment:
nitpick: remove <number> typescript is smart enough to know it's a number

Reply:
Fixed.